File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -340,6 +340,285 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Add-endnotes-in-Word-document).
 
+## Adding a Footnote and Endnote position
+
+Footnotes are typically located at the bottom of a page or beneath text being referenced, and endnotes are typically placed at the end of a document or at the end of a section. This can be done using `FootnotePosition` API and `EndnotePosition` API.
```

Comment:
Footnotes are typically located at the bottom of a page or beneath the text being referenced, and endnotes are typically placed at the end of a document or at the end of a section. This can be done using the `FootnotePosition` API and `EndnotePosition` API.